modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/js/diff.js

```diff
@@ -51,12 +51,12 @@ $('.container-fluid').resize(function(){
  */
 function setViewPanelsHeight(){
     if($(window).width() > 768){
-        $(viewPanel).css('height', ($('body').height() - appBarHeight));
+        $(viewPanel).css('height', ($('body').height() - ($(window).height()*.1)));
```

Comment:
Please explain if and else part separately. Other than that better if you can create a constant for '768' or else use the comment to explain what it is. Alien would feel hard to maintain the code.